src/server/plugins/scan_mixin.js

```diff
@@ -53,10 +54,20 @@ export async function scanMixin(kbnServer, server, config) {
     })
   );
 
-  kbnServer.pluginSpecs = await spec$
-    .merge(logging$.ignoreElements())
```

Comment:
Do we not need to ignore the `logging$` elements anymore?

Reply:
Nope, we're ignoring all of the elements from the final observable and relying on the side-effects in the `.do()` blocks to update `kbnServer.pluginSpecs` and `kbnServer.disabledPluginSpecs`

Reply:
Alternatively we could do something like:


```js
const [enabledSpecs, disabledSpecs] = await Observable
  .combineLatest(spec$.toArray(), disabledSpec$.toArray(), logging$.takeLast(1).defaultIfEmpty())
  .toArray();

kbnServer.pluginSpecs = enabledSpecs;
kbnServer.disabledPluginSpecs = disabledSpecs;
```

or

```js
const [enabledSpecs, disabledSpecs] = await Promise.all([
  spec$.toArray().toPromise(),
  disabledSpec$.toArray().toPromise(),
  logging$.toPromise()
]);

kbnServer.pluginSpecs = enabledSpecs;
kbnServer.disabledPluginSpecs = disabledSpecs;
```

but I'm partial to the individual instructions combined with `Observable.merge()`